test/data/GeomIndex/GeomIndex23.ili

```diff
@@ -0,0 +1,25 @@
+INTERLIS 2.3;
+
+MODEL GeomIndex23
+  AT "mailto:ce@eisenhutinformatik.ch" VERSION "2021-11-25" =
+
+  DOMAIN
+    Vertex2D = COORD
+      2480000.000 .. 2850000.000 [INTERLIS.m],
+      1070000.000 .. 1310000.000 [INTERLIS.m],
+      ROTATION 2 -> 1;
+
+  TOPIC Test =
```

Comment:
Bitte Geometrie-Typen SURFACE,  MULTICOORD, MULTPOLYLINE, MULTISURFACE und auch Radien (STRAIGHTS, ARCS) ergänzen. Damit die Radien funktionieren bitte auch in Iox2gpkg die Env-Berechnung ergänzen (wo sie noch fehlt: writeGeoPackageBinaryHeader(srsId,**null**);).

Modell ungefähr so:

```
    CLASS ClassA1 =
        Geometrie: SURFACE WITH (STRAIGHTS,ARCS) VERTEX Vertex2D WITHOUT OVERLAPS > 0.001;
    END ClassA1;

   STRUCTURE PointStruktur2D =
      coord: Vertex2D;
   END PointStruktur2D;

   !!@ili2db.mapping=MultiPoint
   STRUCTURE MultiPoint2D =
      points: BAG {1..*} OF PointStruktur2D;
   END MultiPoint2D;
	
    CLASS ClassA2 =
	geom : MANDATORY MultiPoint2D;    	
    END ClassA2;

   STRUCTURE LinieStruktur2D =
      Linie: POLYLINE WITH (STRAIGHTS,ARCS) VERTEX Vertex2D;
   END LinieStruktur2D;

   !!@ili2db.mapping=MultiLine
   STRUCTURE MultiLinie2D =
      Linien: BAG {1..*} OF LinieStruktur2D;
   END MultiLinie2D;
  
    CLASS ClassA3 =
	geom : MANDATORY MultiLinie2D;    	
    END ClassA3;

   STRUCTURE FlaecheStruktur2D =
      Flaeche: SURFACE WITH (STRAIGHTS,ARCS) VERTEX Vertex2D WITHOUT OVERLAPS > 0.001;
   END FlaecheStruktur2D;

   !!@ili2db.mapping=MultiSurface
   STRUCTURE MultiFlaeche2D =
      Flaechen: BAG {1..*} OF FlaecheStruktur2D;
   END MultiFlaeche2D;
  
    CLASS ClassA4 =
	geom : MANDATORY MultiFlaeche2D;    	
    END ClassA4;
```

Reply:
Okay, das ist sicher sinnvoll, den Test gerade für alle möglichen Geometrien durchzuführen

Reply:
ich melde mich gerade wieder, sobald das erledigt ist bzw. falls ich evtl. wegen den Radien kurz Rücksprache nehmen muss

Reply:
siehe https://github.com/claeis/ili2db/pull/442#issuecomment-992189187